### PR TITLE
Heatmap

### PR DESCRIPTION
**Pull request by CalyCherkaoui**
I tried to build a clone of the "the Newsweek" page (https://www.newsweek.com/).
The purpose of this exercise is to learn about Bootstrap and its best practices.
In this project, I used many bootstrap tools, such as:
-> grid
-> flexbox
-> cards
-> navigation bar
->Margin/padding
->Typography

### CODE REVIEW

**Review by men32z — COMMENTED** [review]
## Pull request currently under review by Luis Preza (TSE).

Kindly exercise patience 🐱

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1066 @@
...
+  font-weight: 400 !important;
+  font-style: normal !important;
+  letter-spacing: normal !important;
+  line-height: 1.385em !important;
+  color: #333 !important;
+
+  /* margin-top: 0.625rem; */
+}
+
+.opinion-title-like-font {
```
Comment (men32z) [inline style.css]: please remove commented code

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1066 @@
...
+.black-nofill-features {
+  font-family: "Liberation Sans", sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-style: normal;
+  letter-spacing: normal;
+  line-height: 1.538em;
+  color: #272727;
+  text-transform: capitalize;
+}
```
Comment (men32z) [inline style.css]: this is an example that classes with repeated code that you could split to re-use code.

**Review by men32z — APPROVED** [review]
## Good job!!. Project Approved
congratulations, what a great project!! 
descriptive commits, good README( although I think you have too much screenshots), responsiveness, good attention to details, I can't say much about how to improve it, 
there are few things that you should consider,  but since this was a really good project I will trust that you are going to change them before merge.

- [x] change the link to newspaper is not working
- [x] the space betwen the header and the content is bigger than the original project, you should change it.

### changes from the HTML section
these are explained in the comments.

- [x] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [x] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
    - no repeating of the same rule
    - no ID selectors for multiple elements
    - no `!import` overusing
    - no inefficient selectors
    - no inline styling

BlgS1sdIUAEybkW

### Status Approved

If you have any questions, feel free to send me a message through Slack (@men32z)
I would be happy to help! 👩‍💻

Happy coding 🤓

--Luis Preza

File: style.css (hunk 2 above)
Comment (CalyCherkaoui) [reply style.css]: Thanx for helping me improve my quality of coding :)

**Pull request closed without merging**